### ISSUE TITLE
Duplicate markets on xDai

### ISSUE THREAD

**Issue by Mi-Lan** [protofire/omen-exchange#1517]
- So when i create market with WXDAI and then try to add one more currency in this case xDai in the interface there are two identical markets displayed just with different values which doesn't sound right
![Screenshot 2021-01-08 at 14 28 58](https://user-images.githubusercontent.com/33226956/104021092-7e53f680-51be-11eb-9dcb-fef094444223.png)

**Comment by KadenZipfel** [comment]
So you created one market using wxDai and one using xDai?

**Comment by Mi-Lan** [comment]
> So you created one market using wxDai and one using xDai?

Yeah i created market with wxdai and then with add currency option added xDai or other way around...Cant remember it

**Comment by KadenZipfel** [comment]
> > So you created one market using wxDai and one using xDai?
> 
> Yeah i created market with wxdai and then with add currency option added xDai or other way around...Cant remember it

Odd... Good find!
